js-guide.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Newbie Guide: JavaScript</title>
+  </head>
+  <body>
+    <h1>Newbie Guide: JavaScript</h1>
+    <h2>Brief history of language</h2>
+    <p>JavaScript is an interpreted programming language originally created in 1995 by Brendan Eich. It is known as the language of the web, but it can also be run outside of a web browser.</p>
+
+    <h2>Where is Javascript used? What can I program with it?</h2>
+    <ul>
+      <li>Browser / client-side (websites UI, single-page-application frameworks)</li>
+      <li>Server-side / non-browser runtimes (NodeJS)</li>
+      <li>Additional cross-platform applications (Electron, IoT, anything else)</li>
```

Comment:
Can we add some short examples of each of these? Beginners probably have no idea what Electron or NodeJS are.   And may be unfamiliar with the difference between client-side / server-side code.